### PR TITLE
Docs: Add documentation for the `local` connector

### LINKED ISSUE #1245: Add documentation for the `local` connector
There is currently no documentation in [`connectors`](https://webhint.io/docs/user-guide/concepts/connectors/) about the [`local`](https://github.com/webhintio/hint/tree/master/packages/connector-local) connector.

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #1245

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: docs/user-guide/concepts/connectors.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# Connectors
+
+A `connector` is the interface between the `rule`s and the website
+you are testing. It is responsible for loading the website and exposing
+all the information to `sonarwhal` such as resources, network data, etc.
+
+To configure a connector you need to update your `.sonarwhalrc` file to
```
Comment (molant) [inline docs/user-guide/concepts/connectors.md]: > `.hintrc` 

Here and in other places.

File: docs/user-guide/concepts/connectors.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+* `jsdom`: Your website will be loaded using [`jsdom`][jsdom].
+* `chrome`: Your website will be loaded using Chrome and the Chrome
+  Debugging Protocol. This is one of the `remote-debugging-connector`s
+* `edge`: Your website will be loaded using Edge via the
+  [`edge-diagnostics-adapter`][eda]. You will need to run Windows 10
+  Creators Update or later to use it. This connector will only be
+  installed if you are running on it. There are some known issues so
+  please check the [Edge issues](#edge-issues) section below.
+* `local`: This connector will analyze the files specified (a file
+  or a directory). This connector will use [`jsdom`][jsdom] to
```
Comment (alrra) [inline docs/user-guide/concepts/connectors.md]: Drop this, we are analyzing more the HTML files.

File: docs/user-guide/concepts/connectors.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+* `headers`: the headers used to fetch the resources. By default they are:
+
+```json
+ {
+    "Accept-Language": "en-US,en;q=0.8,es;q=0.6,fr;q=0.4",
+    "Cache-Control": "no-cache",
+    "DNT": 1,
+    "Pragma": "no-cache",
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.2924.87 Safari/537.36"
```
Comment (alrra) [inline docs/user-guide/concepts/connectors.md]: > "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36"

**Comment by alrra** [comment]
> I think you are using an old version.

Yes. @sarvaje You need to modify [this file](https://github.com/webhintio/hint/blob/0d8eba6ecddfa30f4c4ecb9915ca076e8e491472/packages/hint/docs/user-guide/concepts/connectors.md).

**Comment by sarvaje** [comment]
Closing in favor of https://github.com/webhintio/hint/pull/1391

**Pull request closed without merging**